src/NexusMods.Library/AddLocalFileJobWorker.cs

```diff
@@ -0,0 +1,43 @@
+using JetBrains.Annotations;
```

Comment:
Consider adding a "Jobs" and potentially "JobWorkers" project subdirectories to organize these